test/lib/usd/translators/UsdExportUsdPreviewSurfaceTest/UsdExportUsdPreviewSurfaceTest.ma

```diff
@@ -0,0 +1,545 @@
+//Maya ASCII 2020 scene
```

Comment:
Do you still have a copy of Maya 2019 you can use to generate the scene. We need to support that one as well until the next release is shipped.

This will most likely fail the preflight as we have a testsuite running on Maya 2019 to catch these issues.

Reply:
Unfortunately we don't have anything below 2020 available at this time. Should I just modify the CMake test file  to only add the test if it's 2020 or higher?

I forget if there were any significant `.ma` changes from 2019 to 2020, otherwise I'd just do surgery on the file.

Reply:
Yes, this is why test\lib\usd\translators\testUsdExportImportRoundtripPreviewSurface.py builds the scene using scripting to be compatible with Maya 2019. This is a bit involved, but feel free to copy the code if it makes sense.

You can also try this replicated scene from Maya 2019. Hopefully I got it sufficiently right that it would only require minor surgery to work with your unit test.
[UsdExportUsdPreviewSurfaceTest_2019.zip](https://github.com/Autodesk/maya-usd/files/7528994/UsdExportUsdPreviewSurfaceTest_2019.zip)

Reply:
Thanks for the 2019 file, I'll ask our TD to look into updating the tests to be compatible.

Reply:
We've updated the test to procedurally generate the files. We've tested in 2020 and 2022, but haven't verified in 2019